pkg/apiserver/resource_manager/service.go

```diff
@@ -0,0 +1,149 @@
+// Copyright 2023 PingCAP, Inc. Licensed under Apache-2.0.
+
+package resourcemanager
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"go.uber.org/fx"
+
+	"github.com/pingcap/tidb-dashboard/pkg/apiserver/user"
+	"github.com/pingcap/tidb-dashboard/pkg/apiserver/utils"
+	"github.com/pingcap/tidb-dashboard/pkg/tidb"
+	"github.com/pingcap/tidb-dashboard/util/featureflag"
+	"github.com/pingcap/tidb-dashboard/util/rest"
+)
+
+type ServiceParams struct {
+	fx.In
+	TiDBClient *tidb.Client
+}
+
+type Service struct {
+	FeatureResourceManager *featureflag.FeatureFlag
+
+	params ServiceParams
+}
+
+func newService(p ServiceParams, ff *featureflag.Registry) *Service {
+	return &Service{params: p, FeatureResourceManager: ff.Register("resource_manager", ">= 7.1.0")}
+}
+
+func registerRouter(r *gin.RouterGroup, auth *user.AuthService, s *Service) {
+	endpoint := r.Group("/resource-manager")
```

Comment:
I feel it's better to use `_`  (aka `/resource_manager`)to keep consistence with other APIs sytle.